doc/BBCode.md

```diff
@@ -1,208 +1,611 @@
+</tr>
+<tr>
+  <td>[s]strike[/s]</td>
+  <td><strike>strike</strike></td>
+</tr>
+<tr>
+  <td>[o]overline[/o]</td>
+  <td><span class="overline">overline</span></td>
+</tr>
+<tr>
+  <td>[color=red]red[/color]</td>
+  <td><span style="color:  red;">red</span></td>
+</tr>
+<tr>
+  <td>[url=http://www.friendica.com]Friendica[/url]</td>
+  <td><a href="http://www.friendica.com" target="external-link">Friendica</a></td>
+</tr>
+<tr>
+  <td>[img]http://friendica.com/sites/default/files/friendika-32.png[/img]</td>
+  <td><img src="http://friendica.com/sites/default/files/friendika-32.png" alt="Immagine/foto"></td>
+</tr>
+<tr>
+  <td>[img=64x32]http://friendica.com/sites/default/files/friendika-32.png[/img]<br>
+<br>Note: provided height is simply discarded.</td>
+  <td><img src="http://friendica.com/sites/default/files/friendika-32.png" style="width: 64px;"></td>
+</tr>
+<tr>
+  <td>[size=xx-small]small text[/size]</td>
+  <td><span style="font-size: xx-small;">small text</span></td>
+</tr>
+<tr>
+  <td>[size=xx-large]big text[/size]</td>
+  <td><span style="font-size: xx-large;">big text</span></td>
+</tr>
+<tr>
+  <td>[size=20]exact size[/size] (size can be any number, in pixel)</td>
+  <td><span style="font-size: 20px;">exact size</span></td>
+</tr>
+<tr>
+  <td>[font=serif]Serif font[/font]</td>
+  <td><span style="font-family: serif;">Serif font</span></td>
+</tr>
+</table>
+
+### Links
+
+<table class="bbcodes">
+<tr>
+  <th>BBCode</th>
+  <th>Result</th>
+</tr>
+<tr>
+  <td>[url]http://friendica.com[/url]</td>
+  <td><a href="http://friendica.com">http://friendica.com</a></td>
+</tr>
+<tr>
+  <td>[url=http://friendica.com]Friendica[/url]</td>
+  <td><a href="http://friendica.com">Friendica</a></td>
+</tr>
+<tr>
+  <td>[bookmark]http://friendica.com[/bookmark]<br><br>
+#^[url]http://friendica.com[/url]</td>
+  <td><span class="oembed link"><h4>Friendica: <a href="http://friendica.com" rel="oembed"></a><a href="http://friendica.com" target="_blank">http://friendica.com</a></h4></span></td>
+</tr>
+<tr>
+  <td>[bookmark=http://friendica.com]Bookmark[/bookmark]<br><br>
+#^[url=http://friendica.com]Bookmark[/url]<br><br>
+#[url=http://friendica.com]^[/url][url=http://friendica.com]Bookmark[/url]</td>
+  <td><span class="oembed link"><h4>Friendica: <a href="http://friendica.com" rel="oembed"></a><a href="http://friendica.com" target="_blank">Bookmark</a></h4></span></td>
+</tr>
+<tr>
+  <td>[url=/posts/f16d77b0630f0134740c0cc47a0ea02a]Diaspora post with GUID[/url]</td>
+  <td><a href="/display/f16d77b0630f0134740c0cc47a0ea02a" target="_blank">Diaspora post with GUID</a></td>
+</tr>
+<tr>
+  <td>#Friendica</td>
+  <td>#<a href="/search?tag=Friendica">Friendica</a></td>
+</tr>
+<tr>
+  <td>@Mention</td>
+  <td>@<a href="javascript:void(0)">Mention</a></td>
+</tr>
+<tr>
+  <td>acct:account@friendica.host.com (WebFinger)</td>
+  <td><a href="/acctlink?addr=account@friendica.host.com" target="extlink">acct:account@friendica.host.com</a></td>
```

Comment:
It isn't supposed to be local. According to `include/bbcode.php` it performs a WebFinger request to guess the related profile page. On the local accounts, it will easily find it, not sure about remote one. Did you test with a remote Friendica account?